src/main/java/com/google/sps/utils/UserUtils.java

```diff
@@ -14,6 +14,13 @@
 
 package com.google.sps;
 
+
+import java.io.*;
+import javax.servlet.*;
+import javax.servlet.http.*;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
```

Comment:
Are any of these new imports used?

Reply:
You need some of them for the cookie object, ill check which and take some out